gamestonk_terminal/README.md

```diff
@@ -274,9 +274,14 @@ Command|Description
 
 Command|Description
 ------ | ------------
-`load`| load cryptocurrency data
-`view`| load and view cryptocurrency data
+`load`| load cryptocurrency data from coingecko
```

Comment:
Probably a 3 column table with:
Command Description Source would be ideal in this situation!

Reply:
Or even have a new line to distinguish between provider.

Similar to how behavioural analysis commands have been done